src/PhaseSlopeIndex.jl

```diff
@@ -102,18 +103,27 @@ function data2para(
     freqlist::AbstractArray,
     method::String,
     subave::Bool,
+    verbose::Bool
 )
-
-    # We would like to avoid transpose and copying the data!
+    # data dimension
+    if ndims(data) != 2
+        data = squeeze(data)
+        ndims(data) != 2 && throw(DimensionMismatch("data must be a 2D-array!"))
```

Comment:
Won't this error always be thrown?

Reply:
Not if the squeeze has reduced it to 2D.